ocf_datapipes/load/nwp/providers/gfs.py

```diff
@@ -1,50 +1,50 @@
 """Open GFS Forecast data"""
 
 import logging
-from pathlib import Path
-from typing import Union
 
-import pandas as pd
 import xarray as xr
 
+from ocf_datapipes.load.nwp.providers.utils import open_zarr_paths
+
+# import pandas as pd
+
 _log = logging.getLogger(__name__)
 
 
-def open_gfs(zarr_path: Union[Path, str]) -> xr.Dataset:
+def open_gfs(zarr_path) -> xr.DataArray:
     """
-    Opens GFS dataset
+    Opens the GFS data
 
     Args:
-        zarr_path: Path to Zarr(s) to open
+        zarr_path: Path to the zarr to open
 
     Returns:
-        Xarray dataset of GFS Forecasts
+        Xarray DataArray of the NWP data
     """
-
     _log.info("Loading NWP GFS data")
 
-    if "*" in zarr_path:
-        nwp = xr.open_mfdataset(zarr_path, engine="zarr", combine="time", chunks="auto")
-    else:
-        nwp = xr.load_dataset(zarr_path, engine="zarr", mode="r", chunks="auto")
+    # Open data
+    gfs: xr.Dataset = open_zarr_paths(zarr_path, time_dim="init_time_utc")
 
-    variables = list(nwp.keys())
+    # _________________EXTRAPOLATION_________________
 
-    nwp = xr.concat([nwp[v] for v in variables], "channel")
-    nwp = nwp.assign_coords(channel=variables)
+    # _log.info("Imputing step 0 for radiation variables")
```

Comment:
Not sure what you mean, the flattening out of channels is still performed (see lines 45, 47), just in a different way

Reply:
Sorry, I meant, this is fine to leave in for the moment. Often we remove commented out code, as it can make the whole code base look messy. But for this I think we should leave it in for the moment.